ios/brave-ios/Sources/Brave/Frontend/Browser/SearchResultAdClickedInfoBar.swift

```diff
@@ -0,0 +1,139 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import Foundation
+import Shared
+import SnapKit
+import SwiftUI
+import UIKit
+
+private struct SearchResultAdClickedInfoBarUX {
+  static let toastHeight: CGFloat = 100.0
+  static let toastPadding: CGFloat = 10.0
+  static let toastCloseButtonWidth: CGFloat = 20.0
+  static let toastLabelFont = UIFont.preferredFont(forTextStyle: .subheadline)
+  static let toastBackgroundColor = UIColor(braveSystemName: .schemesOnPrimaryFixed)
+  static let learnMoreUrl = "https://support.brave.com/hc/en-us/articles/360026361072-Brave-Ads-FAQ"
+}
+
+class SearchResultAdClickedInfoBar: Toast, UITextViewDelegate {
+  let tabManager: TabManager
+
+  init(tabManager: TabManager) {
+    self.tabManager = tabManager
+
+    super.init(frame: .zero)
+
+    self.tapDismissalMode = .outsideTap
+
+    self.clipsToBounds = true
+
+    toastView.backgroundColor = SearchResultAdClickedInfoBarUX.toastBackgroundColor
+
+    let toastContent = createToastContent(
+      Strings.searchResultAdClickedInfoBarTitle + " "
+    )
+    toastView.addSubview(toastContent)
+    toastContent.snp.makeConstraints { make in
+      make.centerX.equalTo(toastView)
+      make.centerY.equalTo(toastView)
+      make.edges.equalTo(toastView).inset(SearchResultAdClickedInfoBarUX.toastPadding)
+    }
+
+    addSubview(toastView)
+    toastView.snp.makeConstraints { make in
+      make.left.right.height.equalTo(self)
+      self.animationConstraint = make.top.equalTo(self).offset(SimpleToastUX.toastHeight).constraint
```

Comment:
I've chose to use `SimpleToastUX.toastHeight` for an animation constraint, because it is suitable for what we have for dismiss animation in the parent class here: https://github.com/brave/brave-core/blob/master/ios/brave-ios/Sources/Brave/Frontend/Browser/Toast.swift#L84

```
    UIView.animate(
      withDuration: duration,
      animations: {
        self.animationConstraint?.update(offset: SimpleToastUX.toastHeight)
        self.layoutIfNeeded()
      },
...
```

Reply:
I think we're going to have issues with this though since this copy is multiline and will get translated to likely be much taller whereas other toasts typically are capped at a certain height.

Reply:
Agreed, I will try to find a way of animation basing on the actual Toast height, not the predefined constants.

Reply:
Agreed, this will affect starting/ending Toast heights during animation. 
I will try to find a way of animation basing on the actual Toast height, not the predefined constants.

Reply:
Fixed